### PR TITLE
[KEYCLOAK-2833] Migrate the Node.js documentation to the official repositories

### PR DESCRIPTION
**Pull request by abstractj**
@lance @helio-frota @lholmquist @sebastienblanc would you mind to review?

### CODE REVIEW

**Comment by helio-frota** [comment]
LGTM

File: README.md [hunk 1]
```diff
@@ -1,277 +1,16 @@
...
 
-[Keycloak](http://keycloak.jboss.org/) is a standalone authentication
+[Keycloak](http://keycloak.org/) is a standalone authentication
 server, akin to a standalone database.  It provides hooks for federated
 authentication, including authenticating against various social networks
 and OAuth providers (G+, Facebook, etc).
 
 This module makes it simple to implement a Node.js Connect-friendly
-application that uses Keycloak for its authentication and authorization needs.
+application that uses Keycloak for its authentication and authorization needs. For usage instructions, please read the [documentation](https://keycloak.gitbooks.io/securing-client-applications-guide/content/v/2.2/topics/oidc/oidc-overview.html).
```
Comment (lance) [inline README.md]: The link here points to the official KC documentation. But that documentation does not include what has been removed from this README. What's there is info about the browser based KC adapter. Shouldn't all of this documentation below be moved to the gitbooks location?

Comment (lance) [inline README.md]: Cool - otherwise, this LGTM

File: README.md [hunk 2]
```diff
@@ -1,277 +1,16 @@
...
```
Comment (stianst) [inline README.md]: URL should updated to (will be available once 2.3 is out):
https://keycloak.gitbooks.io/securing-client-applications-guide/content/topics/oidc/nodejs-adapter.html

**Comment by abstractj** [comment]
Landed

**Pull request closed without merging**